cmd/libs/go2idl/types/types.go

```diff
@@ -72,6 +72,9 @@ const (
 	DeclarationOf Kind = "DeclarationOf"
 	Unknown       Kind = ""
 	Unsupported   Kind = "Unsupported"
+
+	// Protobuf is protobuf type.
+	Protobuf Kind = "Protobuf"
```

Comment:
Oh, I forgot about that.  Yeah, this is questionable, but that's the existing code (not introduced here).  

The goal was to allow two type systems to coexist - if a name identified either what type system (by constant) or type universe (by pointer) it came from that would also accomplish what this was doing.  The protobuf code needed to define imports of type protobuf, as well as identify when conversion between type systems happened in different phases (i.e. when we look at the protobuf struct tag we might make an early type determination about the protobuf type, but otherwise the field was still a Go lang type).  I guess we can restructure the generator to accomplish that - I'll dig into it later this week.